code/models/ContentReviewLog.php

```diff
@@ -40,4 +40,16 @@ public function canView($member = null)
     {
         return (bool) Member::currentUser();
     }
+
+
+    /**
+     * allow the user to edit the fields
+     * @return \FieldList
+     */
+    public function getCMSFields() {
+        $fields = FieldList::create();
+        $fields->push(TextareaField::create('Note'));
+        $fields->push(HiddenField::create('ReviewerID', 'ReviewerID', Member::currentUserID()));
+        return $fields;
+    }
```

Comment:
Thanks for the PR! I'd suggest here that we should utilise the DataObject scaffolder first, then adjust the field list to suit.

E.g.:

```php
$fields = parent::getCMSFields();
$fields->removeByName('ReviewerID');
$fields->push(...
```

Would you mind updating this to suit?